pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -431,7 +431,7 @@ class LightningKokkos(Device):
     # pylint: disable=too-many-instance-attributes
 
     # General device options
-    _device_options = ("rng", "c_dtype", "batch_obs")
```

Comment:
Remove `numpy` random generator from the device option. This is only applicable to MCMC